statics/js/components/topology.js

```diff
@@ -141,31 +141,42 @@ var TopologyComponent = {
           </tab-pane>\
         </tabs>\
         <transition name="slide" mode="out-in">\
-          <div class="left-panel" v-if="currentNode">\
-            <h1>Metadata<span class="pull-right">(id: {{currentNode.id}})\
-              <i v-if="currentNode.group != null" class="node-action fa"\
-                title="Expand/Collapse Node"\
-                :class="{\'fa-expand\': (currentNode.group && currentNode.group.collapsed), \'fa-compress\': (!currentNode.group || !currentNode.group.collapsed)}"\
-                @click="toggleExpandAll(currentNode)"></i></span>\
-            </h1>\
-            <div id="metadata-panel" class="sub-left-panel">\
-              <object-detail :object="currentNodeMetadata"></object-detail>\
-            </div>\
+          <div class="left-panel" v-if="currentNode || currentEdge">\
+            <div v-if="currentNode">\
+              <span v-if="time" class="label center-block node-time">\
```

Comment:
The template need to be reworked to just display metadata

Reply:
do we need to move the tables out from here?